.eslintignore

```diff
@@ -2,3 +2,123 @@ coverage/**
 node_modules/**
 dist/**
 src/index.html
+
+# Python temp file
+*.py[cod]
```

Comment:
Why did each of the `.ignore` files end up with the gitignore content in them?

Reply:
When we run yarn lint and format, I believe it only checks the JS and JSON files (From package.json) so we can include the values in .ignore as it doesn't need to check over these values.

Reply:
Yeah, we don't want to look for `.js` files everywhere in runner root, that was the idea behind it.

Reply:
I'm all for adding more files and directories to avoid it checking things in the root, though the whole ignore seems overkill, did many of these contains js files?